Library/Homebrew/cmd/update.sh

```diff
@@ -390,7 +390,7 @@ EOS
     brew install curl
   fi
 
-  if ! git --version >/dev/null 2>&1
+  if [[ -n "$HOMEBREW_SYSTEM_GIT_TOO_OLD" ]] || ! git --version >/dev/null 2>&1
```

Comment:
@vitorgalvao 👍 to a PR for that!

Reply:
Note that `&>/dev/null` is a bash-ism and is not portable to other shells. In particular, the default shell `/bin/sh` on Ubuntu is Dash, which does not implement `&>/dev/null`. It gets interpreted as redirecting stdout to /dev/null (`>/dev/null`) and backgrounding the process (`&`).

Reply:
@sjackman Best way around that would be to make sure the shell is defined at the top, I’d say. That way there won’t be surprises (especially since macOS will be forever stuck in Bash 3).

But after I’ve made the initial comment, I’ve noticed the rest of this script did already use `&>/dev/null`, so if this change is reverted, we should also make sure to replace those.

Reply:
Fair point. I see now that `bin/brew` is `#!/bin/bash`, so all is good.